src/sync/incomingSyncUtils.js

```diff
@@ -421,6 +475,12 @@ export const createOrUpdateRecord = (database, settings, recordType, record) =>
         name: record.item_name,
         crossReferenceItem: database.getOrCreate('Item', record.cross_ref_item_ID),
         unit: database.getOrCreate('Unit', record.unit_ID),
+        doses: parseNumber(record.doses),
+        isVaccine: parseBoolean(record.boolean),
```

Comment:
Is `record.boolean` correct? I haven't checked the mSupply end, but surely some sore of vaccine-related property name here?

Reply:
haha oops! nope, thanks! 285b03f58cda4382025894656533f442ab413c02